src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,49 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * The DeleteCommand class represents a Command that deletes a
+ * Task from the TaskList.
+ *
+ * @author Edric Yeo
+ */
+public class DeleteCommand extends Command {
+
+    private int idx;
```

Comment:
Could add JavaDoc for this class member to specify that it tracks the index of the task